File: resolucoes/java/lista01/src/main/java/org/example/Ex015.java
/* Crie um programa que leia o número de dias trabalhados
 em um mês e mostre o salário de um funcionário, 
 sabendo que ele trabalha 8 horas por dia e ganha R$25 por hora trabalhada*/

package org.example;

import java.util.Scanner;

public class Ex015 {
    public static void main(String[] args) {
        Ex015.main();

    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Quantos dias de trabalho?: ");
        Integer dias = sc.nextInt();
        System.out.printf("O salário do trabalhador é R$%.2f", dias * 8 * 25);
        sc.close();

    }

}

File: resolucoes/kotlin/lista01/src/main/kotlin/ex015.kt
import java.util.*

internal fun calcularSalario() {
    val scanner = Scanner(System.`in`)
    print("Digite o número de dias trabalhados: ")
    val diasTrabalhados = scanner.nextInt()

    // Define as informações sobre o salário
    val horasPorDia = 8
    val salarioPorHora = 25.0

    // Calcula o salário
    val salario = diasTrabalhados * horasPorDia * salarioPorHora

    // Exibe o salário do funcionário
    println("O salário do funcionário é: R$${salario}")
}

fun main() {
    calcularSalario()
}
